### ISSUE TITLE
Console error in AFFiNE integration

### ISSUE THREAD

**Issue by himself65** [toeverything/blocksuite#3427]
```
index.ts:88 Uncaught TypeError: Cannot read properties of undefined (reading 'schema')
    at index.ts:88:24
    at Array.forEach (<anonymous>)
    at BlockStore._buildSpecMap (index.ts:87:11)
    at BlockStore.applySpecs (index.ts:19:27)
    at BlockSuiteRoot2.connectedCallback (lit-root.ts:47:21)
    at ChildPart._insert (lit-html.ts:1406:53)
    at ChildPart._commitNode (lit-html.ts:1449:36)
    at ChildPart._commitTemplateResult (lit-html.ts:1555:12)
    at ChildPart._$setValue (lit-html.ts:1379:12)
    at render (lit-html.ts:2175:8)
```

Reproduce

```ts
git clone https://github.com/toeverything/AFFiNE.git
cd AFFiNE/apps/docs
yarn install
yarn dev
# open 3000
```

**Comment by doodlewind** [comment]
Seems to be related to block spec, /cc @Saul-Mirone to triage.

**Comment by Saul-Mirone** [comment]
The problem is that your block spec seems wrong, please follow the types here: https://github.com/toeverything/blocksuite/blob/master/packages/blocks/src/preset/index.ts

**Comment by doodlewind** [comment]
This will also be handled by @doouding after discussion, so does the triage of AFFiNE integration issues.

**Comment by doouding** [comment]
There are undefined in the built-in preset array, which may be caused by circular references.

**Comment by mingfang** [comment]
what this fix released?
I'm still getting this problem with version `"@blocksuite/editor": "^0.0.0-20230721134812-6e0e3bef-nightly"
`

**Comment by doodlewind** [comment]
@mingfang could you share about your host environment? Are you using AFFiNE canary, or using BlockSuite in your project? Thanks.

**Comment by mingfang** [comment]
I'm trying to use BlockSuite in SvelteKit.
This is my code
```svelte
<script>
    import '@blocksuite/editor';
    import '@blocksuite/editor/themes/affine.css';

</script>

<simple-affine-editor></simple-affine-editor>
```

**Comment by doodlewind** [comment]
We've fixed pretty much circular deps after #3930 and this doesn't reproduce in our playground, docs and AFFiNE environment for a while. So I'd close this as resolved. Please let me know if any help needed.
